app/benchmarks/README.md

```diff
@@ -20,8 +20,8 @@ The results are outlined in the [results](results.md) document.
 
 ## Key takeaways
 
-We decided to softly limit the number of messages contained in a block, via introducing the `MaxPFBMessages` and `MaxNonPFBMessages`, and checking against them in prepare proposal.
```

Comment:
PrepareProposal is the name of a step so I think it is fine to use "in prepare proposal".